File: PPER.py
number = input("Enter the first and second number with a space between them:\n")
index = number.find(" ")
first_number = int(number[0:index])
second_number = int(number[index+1:])
result = 1

if (first_number>0 and second_number>0):
  for i in range (first_number, first_number-second_number, -1):
    result = (result * i)%1000000
else:
  result = 0
print(result)

File: PPER.cpp
unsigned int partial(int first, int second)
{
  unsigned int x = 1;
  for (unsigned int i = first; i>first-second; i--)
  {
    x = (x*i) %1000000;
  }
  return x;
}

void main()
{
  unsigned int first;
  cin >> first;
  unsigned int second;
  cin >> second;
  unsigned int result;
  if (first>second)
  {
    result = partial (first, second);
  }
  else
  {
    result = partial (second, first);
  }
  cout << result << endl;
}
